.github/workflows/test.yml

```diff
@@ -10,6 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
 
+    env:
+      SECRET_KEY: "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
```

Comment:
Security Issue: The `SECRET_KEY` is hardcoded in the workflow file. This is a security risk as it exposes sensitive information. Consider using GitHub Secrets to securely store and access the `SECRET_KEY`.